index.js

```diff
@@ -12,9 +12,9 @@ const email = document.querySelector('.email');
 const element = email.parentElement;
 const x = element.querySelector('small');
 const form = document.querySelector('.form-input');
-const [fullName, firstName, lastName] = form.elements;
-
+const [fullName, firstName, lastName, message] = form.elements;
 const mediaqueryList = window.matchMedia('(max-width: 768px)');
+const formData = document.querySelector('.form-input'); // gets the form element
 
 const screenTest = (e) => {
   if (e.matches) {
```

Comment:
- [ ] Your javascript functionality works very well. I'd like to point out that your hamburger design doesn't match the Figma design. we want you to match the design as close as possible. you can have the design look like that when you hover or active over the hamburger button. button while it is stationary please leave the design as per the Figma.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/147123767-33b52dbe-9add-428a-81e7-c0df898c752a.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/147123820-c2b57fb6-8342-4fbb-9342-871a9a9e8769.png)